src/Extensions/Features/src/FeatureCollectionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```

Comment:
```suggestion
```

Reply:
I tried but got errors as the project targets .NET Framework as well!